core/app_test.go

```diff
@@ -58,10 +58,11 @@ func TestValidConfigParse(t *testing.T) {
 
 	os.Setenv("TEST", "HELLO")
 	os.Args = []string{"this", "-config", testJSON, "/testdata/test.sh", "valid1", "--debug"}
-	app, err := LoadApp()
+	runnable, err := LoadApp()
```

Comment:
We'll definitely need tests for this new feature. Some things to test:

- passing the `-render` flag with valid arguments, invalid arguments, and without arguments
- that the rendered output is as expected, including that all templating is being applied correctly